src/layout/DwindleLayout.cpp

```diff
@@ -992,6 +992,10 @@ std::any CHyprDwindleLayout::layoutMessage(SLayoutMessageHeader header, std::str
         toggleSplit(header.pWindow);
     } else if (ARGS[0] == "swapsplit") {
         swapSplit(header.pWindow);
+    } else if (ARGS[0] == "movetoroot") {
+        const auto WINDOW = ARGS[1].empty() ? header.pWindow : g_pCompositor->getWindowByRegex(ARGS[1]);
+        const auto STABLE = ARGS[2].empty() || std::stoi(ARGS[2]) != 0;
```

Comment:
std::stoi can throw, needs a try catch.

Also why not just use a keyword like `stable`?